src/algebra/lie/basic.lean

```diff
@@ -181,6 +181,9 @@ instance : has_coe (L₁ →ₗ⁅R⁆ L₂) (L₁ →ₗ[R] L₂) := ⟨morphis
 /-- see Note [function coercion] -/
```

Comment:
I've split the changes in this file into a separate PR, #4912 - in case that makes things easier to review.